shm_wrapper/README.md

```diff
@@ -1,23 +1,31 @@
 # Runtime Shared Memory Wrapper
 
-The Runtime Shared Memory Wrapper provides an API for the device handler, network handler, and executor to use to communicate the state of devices to each other, abstracting away the details of the underlying interprocess communication via shared memory. 
+The Runtime Shared Memory Wrapper provides an API for the device handler, network handler, and executor to use to communicate the state of devices, robot state, and gamepad state to each other, abstracting away the details of the underlying interprocess communication via shared memory. 
 
 ## Contents
 
 `shm_wrapper.h` contains the header file that should be included in each of the processes that use the wrapper. Please read the extensive comments in this file for an overview of the wrapper's usage. Source code for the wrapper is found in `shm_wrapper.c`. These files cannot be compiled or run by themselves; rather, they should be included by the other processes that wish to use it and compiled with those processes.
 
+`shm.c` is the process that is responsible for creating and unlinking all of the semaphores and shared memory blocks that are used by the other Runtime processes. By giving the job of creating and unlinking the shared memory blocks and semaphores to this simply and thus very robust process, it ensures that even if any Runtime process crashes unexpectedly and `systemd` shuts down the processes in some random order, the shared memory blocks will be unlinked u pon Runtime shutdown, thus preventing segmentation faults or other errors upon Runtime restart. To copmile, run
```

Comment:
Simple typos:
* ...and semaphores to this [simple] and thus very robust process, 
* ...the shared memory blocks will be unlinked [upon] Runtime shutdown
* To [compile], run

Reply:
👌